cmd/server/cadence/server_test.go

```diff
@@ -100,9 +99,10 @@ func (s *ServerSuite) TestServerStartup() {
 	lifecycle := fxtest.NewLifecycle(s.T())
 
 	var daemons []common.Daemon
-	services := service.ShortNames(service.List)
+	// Shard distributor should be tested separately
+	services := service.ShortNames(service.List[:len(service.List)-1])
```

Comment:
Lets be more explicit. Removing the last from the list is not a very stable solution.